superset/errors.py

```diff
@@ -47,6 +47,7 @@ class SupersetErrorType(str, Enum):
     CONNECTION_HOST_DOWN_ERROR = "CONNECTION_HOST_DOWN_ERROR"
     CONNECTION_ACCESS_DENIED_ERROR = "CONNECTION_ACCESS_DENIED_ERROR"
     CONNECTION_UNKNOWN_DATABASE_ERROR = "CONNECTION_UNKNOWN_DATABASE_ERROR"
+    CONNECTION_DATABASE_PERMISSIONS_ERROR = "TEST_CONNECTION_DATABASE_PERMISSIONS_ERROR"
```

Comment:
typically it's good to keep the enum name and the value consistent. It's easier to track throughout the code